lib/models/ignored_file.dart

```diff
@@ -11,7 +11,7 @@ class IgnoredFile {
 
   IgnoredFile(this.localID, this.title, this.deviceFolder, this.reason);
 
-  factory IgnoredFile.fromTrashItem(TrashFile trashFile) {
+  factory IgnoredFile.fromFile(File trashFile) {
```

Comment:
If this can be a non-trashed file, the `kIgnoreReasonTrash` should also change?

Reply:
Have refactored this part.